src/Chapter2.hs

```diff
@@ -892,9 +894,10 @@ and reverses it.
   cheating!
 -}
 rewind :: [a] -> [a]
-rewind [] = []
-rewind [x] = [x]
-rewind (x:xs) = rewind xs ++ [x]
+rewind xs = go xs [] where
```

Comment:
Not sure if I understood correctly the go pattern, is that how I was supposed to do it ?